keyboards/skyloong/gk87/q1/ansi/keymaps/default/keymap.c

```diff
@@ -0,0 +1,63 @@
+        KC_LCTL,  KC_LALT,  KC_LGUI,             KC_SPC,   KC_SPC,                    KC_SPC,              KC_RCMD, MO(_MAC_INDEX),  KC_RALT,             KC_RCTL,    KC_LEFT,  KC_DOWN,    KC_RGHT
+    ),
+
+    [_WIN_INDEX] =  LAYOUT_all(
+        _______,             KC_F14,   KC_F15, G(KC_TAB),  KC_WSCH, G(C(KC_S)),   KC_SLEP,   KC_MPRV,  KC_MPLY,    KC_MNXT,   KC_MUTE,  KC_VOLD,     KC_VOLU,    RGB_TOG,
+        _______,  _______,  _______,  _______,  _______,  _______,    TO(_WIN),  TO(_MAC),   _______,  _______,    RGB_TOG,  RGB_RMOD,  RGB_MOD,      KC_DEL,     KC_INS,  KC_HOME,   KC_PGUP,
+        _______,  _______,  _______,  _______,  _______,  _______,     _______,   _______,   _______,  _______,    _______,   _______,  _______,     _______,     KC_DEL,   KC_END,   KC_PGDN,
+        _______,  _______,  _______,  _______,  _______,  _______,     _______,   _______,   _______,  _______,    _______,   _______,               _______,
+        _______,  _______,  _______,  _______,  _______,  _______,     _______,   _______,   _______,  _______,    _______,                          _______,              RGB_VAI,
+        _______,  _______,  _______,            _______,  _______,                _______,             _______,    _______,   _______,               _______,    RGB_SPD,  RGB_VAD,   RGB_SPI
+    ),
+
+    [_MAC_INDEX] = LAYOUT_all(
+        _______,              KC_F1,    KC_F2,    KC_F3,    KC_F4,       KC_F5,     KC_F6,     KC_F7,    KC_F8,      KC_F9,       KC_F10,   KC_F11,   KC_F12,    RGB_TOG,
+        _______,  _______,  _______,  _______,  _______,  _______,    TO(_WIN),  TO(_MAC),   _______,  _______,    RGB_TOG,     RGB_RMOD,  RGB_MOD,   KC_DEL,     KC_INS,  KC_HOME,   KC_PGUP,
+        _______,  _______,  _______,  _______,  _______,  _______,     _______,   _______,   _______,  _______,    _______,      _______,  _______,  _______,     KC_DEL,   KC_END,   KC_PGDN,
+        _______,  _______,  _______,  _______,  _______,  _______,     _______,   _______,   _______,  _______,    _______,      _______,            _______,
+        _______,  _______,  _______,  _______,  _______,  _______,     _______,   _______,   _______,  _______,    _______,                          _______,              RGB_VAI,
+        _______,  _______,  _______,            _______,  _______,                _______,             _______,    _______,      _______,            _______,    RGB_SPD,  RGB_VAD,   RGB_SPI
+    )
+};
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
+    [_WIN] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU)},
+    [_MAC] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU)},
+    [_WIN_INDEX] = { ENCODER_CCW_CW(RGB_VAD, RGB_VAI)},
+    [_MAC_INDEX] = { ENCODER_CCW_CW(RGB_VAD, RGB_VAI)}
+};
+#endif
+
+#if defined(DIP_SWITCH_MAP_ENABLE)
+const uint16_t PROGMEM dip_switch_map[NUM_DIP_SWITCHES][NUM_DIP_STATES] = {
+    DIP_SWITCH_OFF_ON(TO(0), TO(1)),
```

Comment:
It looks like the default layer is what you want here, not "to".  
```suggestion
    DIP_SWITCH_OFF_ON(DF(0), DF(1)),
```